tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -192,17 +194,20 @@ public static void ClearProfilerEnvironmentVariables()
             string profilerEnabled = _requiresProfiling ? "1" : "0";
```

Comment:
How does the new `AutomaticInstrumentationEnabled` field differ from the `_requiresProfiling` field? 🤔 Should we merge them into a single property? If not, should we change this, so we explicitly disable the env vars, rather than not adding them?

```suggestion
            string profilerEnabled = _requiresProfiling && AutomaticInstrumentationEnabled ? "1" : "0";
```

Reply:
I'm not sure how I missed the `_requiresProfiling` instance variable. I didn't see anyone using that field so I just removed it in favor of the new `AutomaticInstrumentationEnabled` property and used it to set profilerEnabled to "1" or "0" as you suggested 👍🏼